### PR TITLE
gsoc: Change output

### PR DESCRIPTION
**Pull request by Nosferatul**

### CODE REVIEW

**Comment by yukiisbored** [comment]
Anyway, I think https://github.com/coala/coala/pull/4369/commits/ae8c45b7e670ed30a6665cd88196d1b1b6b70581 should be squashed to https://github.com/coala/coala/pull/4369/commits/31f8c962de0c8edb1e993f108f77bb23f7fd1bf8

Otherwise, I think this looks good.

**Comment by Adrianzatreanu** [comment]
I agree with yuki, squash them, rest is good.

**Comment by yukiisbored** [comment]
I'm gonna wait for primary mentor @Adrianzatreanu to ack and merge this PR

**Comment by Adrianzatreanu** [comment]
Ok so code is fine but shortlog can be improved, its too abstract. Try smt like: CLI: Improve results output

Also, end sentences in body with period

**Comment by yukiisbored** [comment]
ack https://github.com/coala/coala/pull/4369/commits/ec231bddf7d412ba97ed61481799bfd3d1595c06

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -87,8 +86,11 @@ def highlight_text(no_color, text, lexer=TextLexer(), style=None):
 DIFF_EXCERPT_MAX_SIZE = 4
 
 
-def format_lines(lines, line_nr=''):
-    return '\n'.join('|{:>4}| {}'.format(line_nr, line)
+def format_lines(lines, symbol='', line_nr=''):
+    if symbol == '[':
+        return '\n'.join('{}{:>5}{} {}'.format(symbol, ']', line_nr, line)
```
Comment (yukiisbored) [inline coalib/output/ConsoleInteraction.py]: I think you can do this better.

**Comment by yukiisbored** [comment]
The great gods of CI unplugged one of the wires and made the build fail, I restarted the build and it passed https://travis-ci.org/coala/coala/builds/244904025

**Comment by yukiisbored** [comment]
ack https://github.com/coala/coala/pull/4369/commits/d75d10ab7bb9be4abe7d14f201afc748fb1b96d6

**Comment by yukiisbored** [comment]
I'll give the merge gun to @Adrianzatreanu just in case he want to give it another shot for review

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**